terraform/environments/corporate-staff-rostering/locals_security_groups.tf

```diff
@@ -546,6 +558,470 @@ locals {
+          from_port       = 49152
+          to_port         = 65535
+          protocol        = "UDP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        rpc_dynamic_tcp_web = {
+          description     = "49152-65535: TCP Dynamic Port range"
+          from_port       = 49152
+          to_port         = 65535
+          protocol        = "TCP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+      }
+      egress = {
+        all = {
+          description     = "Allow all egress"
+          from_port       = 0
+          to_port         = 0
+          protocol        = "-1"
+          cidr_blocks     = ["0.0.0.0/0"]
+          security_groups = []
+        }
+      }
+    }
+    app = {
+      description = "New security group for application servers"
+      ingress = {
+        all-from-self = {
+          description = "Allow all ingress to self"
+          from_port   = 0
+          to_port     = 0
+          protocol    = -1
+          self        = true
+        }
+        # IMPORTANT: check if an 'allow all from load-balancer' rule is required
+        # IMPORTANT: check whether http/https traffic is still needed? It's in the original but not used at an app level
+        rpc_tcp_app = {
+          description     = "135: TCP MS-RPC allow ingress from app and db servers"
+          from_port       = 135
+          to_port         = 135
+          protocol        = "TCP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        rpc_tcp_app = {
+          description     = "135: UDP MS-RPC allow ingress from app and db servers"
+          from_port       = 135
+          to_port         = 135
+          protocol        = "UDP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        smb_tcp_app = {
+          description     = "445: TCP SMB allow ingress from app and db servers"
+          from_port       = 445
+          to_port         = 445
+          protocol        = "TCP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        smb_udp_app = {
+          description     = "445: UDP SMB allow ingress from app and db servers"
+          from_port       = 445
+          to_port         = 445
+          protocol        = "UDP"
+          security_groups = ["app", "db"]
+          # NOTE: csr_clientaccess will need to be added here to cidr_blocks
+        }
+        http_2109_csr = {
+          description = "2109: TCP CSR ingress"
+          from_port   = 2109
+          to_port     = 2109
+          protocol    = "TCP"
+          cidr_blocks = ["10.0.0.0/8"]
```

Comment:
Use local.security_group_cidrs.https (or maybe just rename that variable EUC or something so it can be used for all end user compute rules)

Reply:
change https to enduserclient in the 'new' sg code so you have local.security_group_cidrs.enduserclient